extlinks/links/admin.py

```diff
@@ -6,7 +6,8 @@
 
 class LinkEventURLPatternAdminInline(GenericTabularInline):
     model = LinkEvent
-    show_change_link = True
+    # Although not ideal, changing this to False has improved performance
+    show_change_link = False
```

Comment:
Did it improve performance significantly? If not I'd keep the change link for better admin usability.

Reply:
It did. It shaved several seconds of loading time, according to the Django Debug Toolbar

Reply:
I would love to see your local tests to see if it's not a "just in my machine" performance improvement.

Reply:
I think this one is pretty easy to sort out: let's just screenshot the change and ask sam/vipin if the change is okay. If we (engineers) are the only ones using this, then I think it's okay to drop it for the sake of perf. If it's useful on the product side, let's keep it.